lib/iris/fileformats/grib/__init__.py

```diff
@@ -902,30 +895,22 @@ def load_cubes(filenames, callback=None, regularise=True, auto_regularise=None):
 
         .. deprecated:: 1.8. Please use the `regularise` kwarg instead.
```

Comment:
Isn't it hard for people to use `regularise` instead when it doesn't exist? :wink: